src/vcf/vcf_v43.ragel

```diff
@@ -721,8 +721,9 @@
                         ( "AA" | "AC" | "AD" | "ADF" | "ADR" | "AF" | "AN" | "BQ" | "CIGAR" | "DB" | "DP" | "END" | "H2" | "H3" | "MQ" | "MQ0" | "NS" | "SB" | "SOMATIC" | "VALIDATED" | "1000G" ) ;
     info_value      = (print - ";")+ ;
     info_value_list = info_value ("," info_value)* ;
+    info_aa         = (print - ("," | ";" | "="))+ ;
     info_entry      = (
```

Comment:
I think we should try to maintain all 3 version the most similar possible. That will help to keep the differences well highlighted.

 I would introduce here `info_entry_snv` as well. Between versions 4.1 and 4.2 there's no difference here. 4.3 allows spaces (and some punctuation things) in `info_value`, `info_key` and `info_aa`, but otherwise than that I don't see more differences.